tools/Equinox.Tool/Program.fs

```diff
@@ -179,13 +179,14 @@ and [<NoComparison; NoEquality>] TestParameters =
     | [<AltCommandLine "-d">]               DurationM of float
     | [<AltCommandLine "-e">]               ErrorCutoff of int64
     | [<AltCommandLine "-i">]               ReportIntervalS of int
-    | [<CliPrefix(CliPrefix.None); Last>]                      Cosmos   of ParseResults<Storage.Cosmos.Parameters>
-    | [<CliPrefix(CliPrefix.None); Last>]                      Dynamo   of ParseResults<Storage.Dynamo.Parameters>
-    | [<CliPrefix(CliPrefix.None); Last>]                      Es       of ParseResults<Storage.EventStore.Parameters>
-    | [<CliPrefix(CliPrefix.None); Last>]                      Memory   of ParseResults<Storage.MemoryStore.Parameters>
-    | [<CliPrefix(CliPrefix.None); Last; AltCommandLine "ms">] MsSql    of ParseResults<Storage.Sql.Ms.Parameters>
-    | [<CliPrefix(CliPrefix.None); Last; AltCommandLine "my">] MySql    of ParseResults<Storage.Sql.My.Parameters>
-    | [<CliPrefix(CliPrefix.None); Last; AltCommandLine "pg">] Postgres of ParseResults<Storage.Sql.Pg.Parameters>
+    | [<CliPrefix(CliPrefix.None); Last>]                         Cosmos    of ParseResults<Storage.Cosmos.Parameters>
+    | [<CliPrefix(CliPrefix.None); Last>]                         Dynamo    of ParseResults<Storage.Dynamo.Parameters>
+    | [<CliPrefix(CliPrefix.None); Last>]                         Es        of ParseResults<Storage.EventStore.Parameters>
+    | [<CliPrefix(CliPrefix.None); Last>]                         Memory    of ParseResults<Storage.MemoryStore.Parameters>
+    | [<CliPrefix(CliPrefix.None); Last; AltCommandLine "ms">]    MsSql     of ParseResults<Storage.Sql.Ms.Parameters>
+    | [<CliPrefix(CliPrefix.None); Last; AltCommandLine "my">]    MySql     of ParseResults<Storage.Sql.My.Parameters>
+    | [<CliPrefix(CliPrefix.None); Last; AltCommandLine "pg">]    Postgres  of ParseResults<Storage.Sql.Pg.Parameters>
+    | [<CliPrefix(CliPrefix.None); Last; AltCommandLine "msgdb">] MessageDb of ParseResults<Storage.Sql.MsgDb.Parameters>
```

Comment:
maybe `mdb` to differentiate from the two letter aliases being SSS variants?
Maybe change case name to `Message` so it's usable without an alias? (or even leave it as `messagedb`) ? (main objection is to removing vowels as in general that's cryptic. Having said that if MessageDB standard docs and naming uses that contraction, then certainly stay with that idiomatic abbreviaton; just saying that the scheme here is for things to try to balance pithiness with obviousness)